peak/assembler/assembler_abc.py

```diff
@@ -0,0 +1,157 @@
+from abc import ABCMeta, abstractmethod
+from hwtypes import AbstractBitVector, BitVector
+from hwtypes.adt import Enum, Product, Sum, Tuple
+from hwtypes.adt_meta import BoundMeta
+
+import typing as tp
+import warnings
+import weakref
+
+def _issubclass(sub , parent : type) -> bool:
+    try:
+        return issubclass(sub, parent)
+    except TypeError:
+        return False
```

Comment:
This function is defined in both `assembler.py` and `assembler_abc.py`, maybe import one from the other or put it in file like `assembler_util.py`?